src/pymediainfo/__init__.py

```diff
@@ -104,10 +86,28 @@ def __init__(self, xml_dom_fragment: ET.Element):
                     except ValueError:
                         pass
 
-    def __repr__(self):  # type: ignore
+    def __eq__(self, other: object) -> bool:
+        if not isinstance(other, Track):
+            return False
+        return self.__dict__ == other.__dict__
+
+    def __getattribute__(self, name: str) -> Any:
+        try:
+            return object.__getattribute__(self, name)
+        except AttributeError:
+            pass
+        return None
+
+    def __getstate__(self) -> dict[str, Any]:
+        return self.__dict__
+
+    def __setstate__(self, state: dict[str, Any]) -> None:
+        self.__dict__ = state
+
+    def __repr__(self) -> str:
         return "<Track track_id='{}', track_type='{}'>".format(self.track_id, self.track_type)
 
-    def to_data(self) -> Dict[str, Any]:
+    def to_data(self) -> dict[str, Any]:
```

Comment:
Yes, but because this import is done at the top of the file, `from __future__ import annotations`, type annotations are not processed at run time, only by type checkers. So it will also run in python 3.8.

Reply:
You dropped Python 3.8 in any case so I don't think it matters if it breaks :)